// -------------------------- Try... Catch block
// try{
//   let x;
//   console.log(x);
//   throw new Error(`variable is not undefined`)
// }
// catch(err){
//   console.log(err)
// }


// -------------------------- Try...Catch to throw oen errors
// function double(num){
//   if(isNaN(num)){
//     throw new Error(`${num} is not a number !!!`)
//   }
//   return num * 2;
// }
// try{
//   const y = double('1');
//   console.log(y)
// }
// catch(err){
//   console.log(err)
// }


